src/tests/test_salute.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+import pytest
+
+from ..service.salute import SaluteService
```

Comment:
Please remove the newline above, add one below, and remove the relative import
```suggestion
from service.salute import SaluteService

```